ios/RNSentry.m

```diff
@@ -140,19 +140,23 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSLog(@"Bridge call to: deviceContexts");
+    NSMutableDictionary<NSString *, id> *contexts = [NSMutableDictionary new];
     // Temp work around until sorted out this API in sentry-cocoa.
     // TODO: If the callback isnt' executed the promise wouldn't be resolved.
     [SentrySDK configureScope:^(SentryScope * _Nonnull scope) {
         NSDictionary<NSString *, id> *serializedScope = [scope serialize];
         // Scope serializes as 'context' instead of 'contexts' as it does for the event.
-        NSDictionary<NSString *, id> *contexts = [serializedScope valueForKey:@"context"];
+        NSDictionary<NSString *, id> *tempContexts = [serializedScope valueForKey:@"context"];
+        if (tempContexts != nil) {
+            [contexts addEntriesFromDictionary:tempContexts];
+        }
 #if DEBUG
         NSData *data = [NSJSONSerialization dataWithJSONObject:contexts options:0 error:nil];
         NSString *debugContext = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
         NSLog(@"Contexts: %@", debugContext);
 #endif
-        resolve(contexts);
     }];
+    resolve(contexts);
```

Comment:
`m`: Did you move this out of the callback on purpose?

Reply:
yes, otherwise the Promise never gets resolved, the `configureScope` callback isn't called if the SDK isn't init'ed.